meta-mender-qemu-community/recipes-bsp/uefi/edk2-firmware_%.bbappend

```diff
@@ -0,0 +1,24 @@
+export GCC5_ARM_PREFIX = "${TARGET_PREFIX}"
+
+export CLANG38_ARM_PREFIX = "${TARGET_PREFIX}"
+
+#FIXME - arm32 doesn't work with clang due to a linker issue
```

Comment:
Come again?

Reply:
What would you like to know? This refers to some internals of `meta-arm`, which is a layer dependency. The snippet was provided by a community member.

Reply:
Just made me curious as to what it was :)